crates/bevy_render/src/lib.rs

```diff
@@ -61,12 +61,11 @@ pub mod prelude {
         },
         render_resource::Shader,
         spatial_bundle::SpatialBundle,
-        texture::{Image, ImagePlugin, IntoDynamicImageError},
```

Comment:
`Image` and `IntoDynamicImageError` should be added to a new `bevy_image::prelude` module, and that prelude should be included by the main `bevy::prelude` (just like all the other member crates) to avoid some of the changes in the examples.

Reply:
Ok. Just those two items? I was thinking that `BevyDefault` is one of the more widely used items and could also be in a prelude.

Reply:
That seems reasonable. I only mentioned those two since they are currently a part of the `bevy_render` prelude. Since `BevyDefault` is used a lot it seems reasonable to include too, especially with the `as _` trick.